frontend/src/components/Sidebar.tsx

```diff
@@ -23,6 +23,11 @@ export const Sidebar = ({
   const location = useLocation()
   const [showHistory, setShowHistory] = useState<boolean>(false)
 
+  const signOut = () => {
+    document.cookie = "auth-token=; expires=Thu, 01 Jan 1970 00:00:00 UTC; path=/;"
+    window.location.href = "/auth"
```

Comment:
Are we sure this will trigger the router?

Reply:
<img width="811" alt="Screenshot 2025-02-24 at 12 35 55 PM" src="https://github.com/user-attachments/assets/8584c595-baa3-4a69-8817-a8c6c81eaff2" />